source/manual/dns.html.md

```diff
@@ -45,39 +45,106 @@ The deployment process is complex, so it's much easier to [use the automation pr
+1. Get a personal access token for GitHub by navigating to the [personal access token](https://github.com/settings/tokens) page then generate a new token with the default permissions. Set this as an environment variable, alongside your GitHub username:
 
-USAGE:
-   gds govuk dns [command options] [arguments...]
+    ```
+    export GITHUB_TOKEN=<token_from_github_ui>
+    export GITHUB_USERNAME=<your_github_username>
+    ```
 
-OPTIONS:
-...
-```
+1. Check you can access AWS (in production as a poweruser) using the following, if not, follow the steps in the [Get Started documentation](/manual/get-started.html#8-get-aws-access).
+
+    ```
+    gds aws govuk-production-poweruser aws s3 ls
+    ```
 
-For example, to have Jenkins run a terraform plan of changes to service.gov.uk against AWS, run:
+1. Run a plan in AWS:
 
-```
-$ gds govuk dns --provider aws --zone service.gov.uk --action plan --role govuk-production-poweruser
-```
+    ```
+    gds govuk dns --provider aws --zone service.gov.uk --action plan --role govuk-production-poweruser
+    ```
+
+1. Navigate to [the production Jenkins](https://deploy.blue.production.govuk.digital/job/Deploy_DNS/) and check the output of the job. A typical change where you replace nameservers with new ones will look like the following (check the name is the correct subdomain for your change and that the old and new nameservers are correct):
+
+    ```
+    ~ resource "aws_route53_record" "NS_passport" {
+          id                               = "Z2QHB5HOZRWMYP_passport_NS"
+          name                             = "passport"
+        ~ records                          = [
+            - "ns1.p08.dynect.net.",
+            - "ns2.p08.dynect.net.",
+            - "ns3.p08.dynect.net.",
+            - "ns4.p08.dynect.net.",
+            + "ns-1435.awsdns-51.org.",
+            + "ns-144.awsdns-18.com.",
+            + "ns-1559.awsdns-02.co.uk.",
+            + "ns-535.awsdns-02.net.",
+          ]
+          # (6 unchanged attributes hidden)
+      }
+    ```
+
+    > It's normal to see changes in TXT records relating to escaping of quotes. You can safely ignore these if they don't change any of the content of the record. This is a bug in the way we handle splitting long TXT records between AWS and GCP in our [YAML -> Ruby -> Terraform process](https://github.com/alphagov/govuk-dns).
+
+1. Install gcloud:
+
+    ```
+    brew install --cask google-cloud-sdk
+    source "$HOMEBREW_PREFIX/Caskroom/google-cloud-sdk/latest/google-cloud-sdk/path.zsh.inc"
+    ```
 
-This will trigger a build in Jenkins. Check the build output to see what terraform has done / would do.
+1. Login to the gcloud CLI by running the followng, then logging into your GDS Google account in the browser window that will open:
 
-### Deploying a change
+    ```
+    gcloud auth login
+    ```
+
+1. Run a plan in GCP:
+
+    ```
+    gds govuk dns --provider gcp --zone service.gov.uk --action plan --role govuk-production-poweruser
```

Comment:
Perhaps a line to say that running a plan is the equivalent of testing it, and no actual changes are made until the "apply" stage?

Reply:
Good idea.  I'll add this.